apps/analysis/tasks.py

```diff
@@ -51,6 +52,7 @@ def trigger_automatic_summary(_id):
         Entry.objects.filter(
             project=a_summary.project,
             id__in=a_summary.entries_id,
+            lead__confidentiality=Lead.Confidentiality.UNPROTECTED,
```

Comment:
```suggestion
            # NOTE: NLP is using LLM, to avoid data leakage we only pass UNPROTECTED
            lead__confidentiality=Lead.Confidentiality.UNPROTECTED,
```